tests/scripts/check-files.py

```diff
@@ -138,7 +138,8 @@ def __init__(self):
         super().__init__()
         self.heading = "TODO present:"
         self.files_exemptions = [
-            __file__, "benchmark.c", "pull_request_template.md"
```

Comment:
Why not replace `check-files.py` by `__file__`?

Also the commit message is misleading. `check-files.py` was already not checking _itself_. What this commit does is to avoid checking files with the same name, for example if there is a copy of `check-files.py` in a subdirectory or if you run `check-files.py` from outside.

Reply:
I'd suggest changing `__file__` to `os.path.basename(__file__)`. That lets you keep not needing to worry about the file name, while still excluding all files with the same name.